solutions/usaco-916.mdx

```diff
@@ -66,6 +66,40 @@ public class Revegetation {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+MAX_PASTURE = 151
+	
+a, b, g = [0] * MAX_PASTURE, [0] * MAX_PASTURE, [0] * MAX_PASTURE
+	
+with open('revegetate.in') as read:
+	n, m = map(int, read.readline().split())
+	
+	for i in range(m):
+		a[i], b[i] = map(int, read.readline().split())
+		if a[i] > b[i]:
+			a[i], b[i] = b[i], a[i]
+
+ans = []
+
+for i in range(1, n + 1):
+	for j in range(1, 5):
```

Comment:
put `range(1, 5)` in a constant labelled GRASS_TYPES

Reply:
okay